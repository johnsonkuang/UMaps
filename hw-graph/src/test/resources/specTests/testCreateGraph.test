# Test Create Graph
# Tests that a graph can be created

# Create a graph
CreateGraph graph1