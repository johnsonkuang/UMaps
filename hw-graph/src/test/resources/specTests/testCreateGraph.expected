# Test Create Graph
# Tests that a graph can be created

# Create a graph
created graph graph1